components/backends/mocker/src/dynamo/mocker/main.py

```
@@ -19,35 +23,96 @@
-    engine_config = await make_engine(runtime, entrypoint_args)
-    await run_input(runtime, args.endpoint, engine_config)
+    
+    # Handle extra_engine_args: either use provided file or create from CLI args
+    if args.extra_engine_args:
+        # User provided explicit JSON file
+        extra_engine_args_path = args.extra_engine_args
+        logger.info(f"Using provided MockEngineArgs from {extra_engine_args_path}")
+    else:
+        # Create temporary JSON file from CLI arguments
+        extra_engine_args_path = create_temp_engine_args_file(args)
+        logger.info(f"Created MockEngineArgs from CLI arguments")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)